src/App.tsx

```diff
@@ -1,10 +1,47 @@
+/* eslint-disable quote-props */
 /* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Try to rewrite the code so that it is not necessary to disable the linter

Reply:
not fixed